### ISSUE TITLE
Fix shopping button not showing up in view hierarchy

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#16443]
Currently the shopping button is not showing up in the view hierarchy in UITests and also can’t be selected with the accessibility inspector.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-7413)

**Comment by anna-yeddi** [comment]
Since the issue is likely to be caused by missing or improper role provided to the control, marking this as access-S2. It needs to be tested to make sure not only VoiceOver can navigate to and activate this button, but also Switch Control and Voice Control too, otherwise this may create a blocker for a user.

Please do not hesitate to reach out if you have questions with the remediation

**Comment by data-sync-user** [comment]
➤ Winnie Teichmann commented:

For QA: Make sure shopping button is reachable using voice over and voice control.

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Verified as fixed on v9000 (34959) with iPhone 13 Pro (15.7.1).

Please note that I was able to access all the buttons on the screen regarding the review quality check bottom sheet but on all the products when expanding it, if taping through the lines, FF reads everything that is on the webpage behind the review quality check bottom sheet.

I added https://github.com/mozilla-mobile/firefox-ios/issues/16762 ( https://github.com/mozilla-mobile/firefox-ios/issues/16762|smart-link ) for that issue.
